fw/apm.c

```diff
@@ -1260,14 +1275,22 @@ tfw_apm_hm_srv_alive(int status, TfwStr *body, void *apmref)
 		return true;
 	}
 
+	if (unlikely(tfw_body_iter_init(&it, &chunk, body->data, body->skb,
+					skb_head)))
+	{
+		T_WARN_NL("Invalid body. Health monitor '%s': status '%d' \n",
+			  hm->name, status);
+		return false;
+	}
+
 	/*
 	 * Special case for 'auto' monitor: generate crc32
 	 * from body of first response and store it into monitor.
 	 */
 	if (!hm->crc32 && hm->auto_crc) {
-		hm->crc32 = tfw_str_crc32_calc(body);
```

Comment:
tfw_str_crc32_calc is now unused, we can remove it from our code and tests

Reply:
I would prefer don't remove this function. Time to time we use crc32 for `TfwStr` even for debug purposes and it useful to have such function.